css/teams/5/blog-posts.css

```diff
@@ -0,0 +1,655 @@
+padding-right: 7em;
+display: flex;
+justify-content: space-evenly;
+align-items: flex-start;
+}
+
+.features-title{
+  color: var(--color-gray-900);
+  display: block;
+  margin-top: 2em;
+}
+
+.title-content {
+  color: var(--color-gray-500);
+}
+
+.direction-col p {
+  color: var(--color-gray-500);
+}
+
+.ut-action-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  background-color: var(--color-primary-800);
+  box-shadow: 0px 20px 24px -4px rgba(16, 24, 40, 0.1), 0px 8px 8px -4px rgba(16, 24, 40, 0.04);
+  border-radius: 24px;
+  margin-left: 7em;
+  margin-right: 7em;
+  background-image: url(../../../teams/5/assets/img.svg);
+  background-position: top right;
+  height: 400px;
+  background-repeat: no-repeat;
+}
+
+
+.ut-action-container-inner {
+display: flex;
+flex-direction: column;
+align-items: flex-start;
+justify-content: center;
+padding: 64px;
+width: 736px;
+height: 310px;
+}
+
+.ut-action-container-inner-text{
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+
+.features-2{
+  padding-top: 6em;
+  padding-bottom: 6em;
+  padding-left: 7em;
+  padding-right: 7em;
+  background-image: url(../../../teams/5/assets/img2.png);
+  background-repeat: no-repeat;
+  background-position: bottom right;
+  background-size: 45%;
+}
+
+
+.ut-large-title{
+  margin: 0;
+  margin-top: 0.5em;
+}
+
+.ut-features-2-textbox{
+  /* margin: 40px; */
+  display: flex;
+  flex-direction: column;
+  padding: 0px;
+  position: static;
+
+  height: 160px;
+
+}
+
+.ut-learn-more{
+  font-family: Inter;
+font-style: normal;
+font-weight: 500;
+font-size: 16px;
+line-height: 24px;
+/* identical to box height, or 150% */
+
+/* Primary/700 */
+color: #6941C6;
+}
+
+.ut-a-tag{
+   text-decoration: none;
+}
+
+.img2{
+ width: 576px;
+ height: 512px;
+}
+
+.pricing {
+  padding-top: 6em;
+  padding-bottom: 6em;
+  padding-left: 7em;
+  padding-right: 7em;
+  background: var(--color-gray-50);
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  justify-content: space-between;
+}
+
+.card-wrapper {
+  display: flex;
+  flex-direction: row;
+  column-gap: 2em;
+}
+
+.card-wrapper .ut-text-md-reg {
+  color: var(--color-gray-500);
+}
+
+.card{
+ display: flex;
+ flex-direction: column;
+ align-items: center;
+ /* padding: 10px;
+ position: static; */
+ width: 336px;
+ height: 582px;
+
+/* White */
+ background: #FFFFFF;
+/* Gray/200 */
+ border: 1px solid var(--color-gray-200);
+ box-sizing: border-box;
+/* Shadow/lg */
+ box-shadow: 0px 12px 16px -4px rgba(16, 24, 40, 0.1), 0px 4px 6px -2px rgba(16, 24, 40, 0.05);
+ border-radius: 16px;
+}
+
+.card ul {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  list-style: none;
+  /* padding: 0; */
+  padding-top: 2em;
+  padding-bottom: 2em;
+}
+
+.card button{
+  width: 100%;
+  height: 3em;
+  border:none;
+  border-radius: 0.5em;
+  color: white;
+  background-color: var(--color-primary-600);
+}
+
+.row {
```

Comment:
do not use row as a class name use classes that are more related to the responsibility of the elements in the page